lib/simplecov.rb

```diff
@@ -203,7 +203,10 @@ def run_exit_tasks!
 
       # Force exit with stored status (see github issue #5)
       # unless it's nil or 0 (see github issue #281)
-      Kernel.exit exit_status if exit_status && exit_status > 0
+      if exit_status && exit_status > 0
+        $stderr.printf("SimpleCov failed build with exit %d", exit_status)
```

Comment:
Not sure I like the wording here - it's not always a build I run simplecov locally all the time. How about we remove the `build` and jsut say `SimpleCov failed with exit %d` ?